tests/testthat/test-1-module_1-defaults.R

```diff
@@ -0,0 +1,255 @@
+
+## Set up ----
+
+  if (!testthat::is_testing()){
+    library(testthat)
+    source(file.path(testthat::test_path(), "setup.R"))
+  }
+
+  # Set project path
+  projectPath <- file.path(testDirs$outputs, "1-module_1-defaults")
+  dir.create(projectPath)
+
+  # Set inputs
+  inputObjects <- moduleInputs
+
+
+## Run simInit and spades ----
+
+  sims <- list()
+
+  test_that("simInit runs successfully", {
+
+    sims$init <<- withCallingHandlers(suppressMessages(
```

Comment:
superassignment operator used here to allow for `simInit` and `spades` to be run inside `test_that`, but for the outputs to be individually checked inside their own `test_that` calls. This seemed better organized than to have everything checked in one `test_that` call. Any foreseeable issues with this? A better way to do it?

Reply:
Full tests moved into a single `test_that` call: https://github.com/PredictiveEcology/CBM_dataPrep_SK/pull/10/commits/d7866ad7d085af1a821252fbbe2ee9d11e4f63de